src/components/inputs/async-creatable-select-input/async-creatable-select-input.spec.js

```diff
@@ -18,6 +18,9 @@ class TestComponent extends React.Component {
             props,
             ...rest
           ),
+    loadOptions: PropTypes.func,
+    defaultOptions: PropTypes.bool,
```

Comment:
Is this a bool or object?

Reply:
it was set as a bool in the existing defaultProps

Reply:
It can either be an array of objects (options) or a boolean, see [`react-select` docs](https://react-select.com/props#async-props).